agent/bindir/rolling-maintenance.in

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
On line 79 and 83/84 - the error is thrown  "Unexpected error on systemd service: write() argument must be str, not bytes".

This needs to be fixed, and also, the output given in the API and also inside the rolling-maintenance.log (on the KVM host) is prepended with letter **b**, and appended with    **\n**

Reply:
i.e. 20:50:23,239 rolling-maintenance INFO **b**'preflight stage' (from the log)
or
"output": "**b**'This is Maintenance Stage'**\n**", (from the API output)

Reply:
Latest commit should solve this